data/human/human missions.txt

```diff
@@ -6849,3 +6849,308 @@ mission "FW Refugees to Humanika"
+			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
+			`	You go back to the <ship>'s landing pad and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
+			choice
+				`	"I am Captain <first> <last>, well met."`
+				`	"Hi, I'm <first>."`
+					goto loadup
+			`	They all frown at your formality.`
+
+			label loadup
+			`	"Good. Now that that's done with, let's load up," says Bunker. "Taking down the Man. Whoo, I love it!"`
+				accept
+
+			label out
+			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`
+				decline
+
+	on visit
+		dialog `The filming equipment is aboard an escort. Better take off and wait for it to arrive.`
+
+
+mission "Adelita 3"
+	landing
+	name "Film slave transaction"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
+	source "Maker"
+	destination "Hippocrates"
+	"apparent payment" 200000
+	infiltrating
+	to offer
+		has "Adelita 2: done"
+	passengers 1
+	cargo "stealth drones" 1
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck, and the two walk off into the spaceport. You and Bunker go back to the <ship> and prepare to take off for <planet>.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+mission "Adelita 4"
+	landing
+	name "Slave trade"
+	description "Pick up the team on <destination>. (You will be paid <payment> after you complete the entire job.)"
+	source "Hippocrates"
+	destination "Maker"
+	"apparent payment" 200000
+	to offer
+		has "Adelita 3: done"
+	passengers 1
+	cargo "stealth drones" 1
+
+	on offer
+		conversation
+			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but as you are well within the atmosphere Bunker releases the drones from the open cargo bay without needing a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`
```

Comment:
```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on auto-pilot to the rendezvous site."`
```

Reply:
```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```